Replace symbolSet with symbolImage in CarPlay maneuver. (#4161)
mapbox/mapbox-navigation-ios: CHANGELOG.md

* `NavigationMapView.removeAlternativeRoutes()` and `NavigationMapView.removeContinuousAlternativeRoutesDurations()` were made public to provide a way to remove previously shown alternative routes and alternative routes duration annotations, respectively. ([#4134](https://github.com/mapbox/mapbox-navigation-ios/pull/4134))

> ### Guidance Instructions

Reviewer: Nit: the release script requires sentence case.

```suggestion
### Guidance instructions
```

> * Added `VisualInstruction.maneuverImage(side:userInterfaceStyle:)` to generate maneuver image for iOS 13 and above versions. ([#4161](https://github.com/mapbox/mapbox-navigation-ios/pull/4161))

Reviewer: ```suggestion
* Added the `VisualInstruction.maneuverImage(side:userInterfaceStyle:)` method to generate a maneuver image on iOS 13 and above. ([#4161](https://github.com/mapbox/mapbox-navigation-ios/pull/4161))
```
